src/patternfly/components/Card/card.scss

```
@@ -1,150 +1,152 @@
+
+.#{$card} {
+  --#{$card}--BackgroundColor: var(--#{$pf-global}--BackgroundColor--100);
+  --#{$card}--BoxShadow: var(--#{$pf-global}--BoxShadow--sm);
+  --#{$card}--first-child--PaddingTop: var(--#{$pf-global}--spacer--lg);
+  --#{$card}--child--PaddingRight: var(--#{$pf-global}--spacer--lg);
+  --#{$card}--child--PaddingBottom: var(--#{$pf-global}--spacer--lg);
+  --#{$card}--child--PaddingLeft: var(--#{$pf-global}--spacer--lg);
+  --#{$card}--c-divider--child--PaddingTop: var(--#{$pf-global}--spacer--lg);
+  --#{$card}__title--not--last-child--PaddingBottom: var(--#{$pf-global}--spacer--md);
+  --#{$card}__title-text--FontFamily: var(--#{$pf-global}--FontFamily--heading--sans-serif);
+  --#{$card}__title-text--FontSize: var(--#{$pf-global}--FontSize--md);
+  --#{$card}__title-text--FontWeight: var(--#{$pf-global}--FontWeight--normal);
+  --#{$card}__title-text--LineHeight: var(--#{$pf-global}--LineHeight--md);
+  --#{$card}__body--FontSize: var(--#{$pf-global}--FontSize--md);
+  --#{$card}__footer--FontSize: var(--#{$pf-global}--FontSize--md);
+  --#{$card}__actions--PaddingLeft: var(--#{$pf-global}--spacer--md);
+  --#{$card}__actions--Gap: var(--#{$pf-global}--spacer--sm);
+  --#{$card}__actions--MarginTop: calc(var(--#{$pf-global}--spacer--form-element) * -1);
+  --#{$card}__actions--MarginBottom: calc(var(--#{$pf-global}--spacer--form-element) * -1);
 
   // Expandable
-  --pf-c-card__header-toggle--MarginTop: calc(var(--pf-global--spacer--form-element) * -1);
-  --pf-c-card__header-toggle--MarginRight: var(--pf-global--spacer--xs);
-  --pf-c-card__header-toggle--MarginBottom: calc(var(--pf-global--spacer--form-element) * -1);
-  --pf-c-card__header-toggle--MarginLeft: calc(var(--pf-global--spacer--md) * -1);
-  --pf-c-card__header-toggle-icon--Transition: var(--pf-global--Transition);
-  --pf-c-card--m-expanded__header-toggle-icon--Rotate: 90deg;
+  --#{$card}__header-toggle--MarginTop: calc(var(--#{$pf-global}--spacer--form-element) * -1);
+  --#{$card}__header-toggle--MarginRight: var(--#{$pf-global}--spacer--xs);
+  --#{$card}__header-toggle--MarginBottom: calc(var(--#{$pf-global}--spacer--form-element) * -1);
+  --#{$card}__header-toggle--MarginLeft: calc(var(--#{$pf-global}--spacer--md) * -1);
+  --#{$card}__header-toggle-icon--Transition: var(--#{$pf-global}--Transition);
+  --#{$card}--m-expanded__header-toggle-icon--Rotate: 90deg;
 
 
   // Selectable/Clickable
-  --pf-c-card--m-selectable--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --#{$card}--m-selectable--BorderWidth: var(--pf-global--BorderWidth--sm);
 
   // Selected state (checked) of selectable card
-  --pf-c-card--m-selectable--m-selected--BackgroundColor: var(--pf-global--BackgroundColor--200);
-  --pf-c-card--m-selectable--m-selected--BorderColor: var(--pf-global--active-color--100);
+  --#{$card}--m-selectable--m-selected--BackgroundColor: var(--pf-global--BackgroundColor--200);
```

Looks like there are a handful of vars that need to reference `$pf-global` in this stylesheet

```suggestion
  --#{$card}--m-selectable--m-selected--BackgroundColor: var(--#{$pf-global}--BackgroundColor--200);
```